recipes/xmlsec/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+from conans import ConanFile, tools, AutoToolsBuildEnvironment
+import os
+import glob
+
+class XmlSecConan(ConanFile):
+    name = "xmlsec"
+    description = "XML Security Library is a C library based on LibXML2. The library supports major XML security standards."
+    license = ("MIT", "MPL-1.1")
+    homepage = "https://github.com/lsh123/xmlsec"
+    url = "https://github.com/conan-io/conan-center-index"
+    generators = "pkg_config"
+    settings = "os", "compiler", "arch", "build_type"
+    topics = ("xml", "signature", "encryption")
+    options = {
+        "shared": [True, False], 
+        "fPIC": [True, False],
+        "with_xslt": [True, False]
+        }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "with_xslt": False
+        }
+    
+    _autotools = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def requirements(self):
+        self.requires("libxml2/2.9.10")
+        self.requires("openssl/1.1.1g")
```

Comment:
```suggestion
        self.requires("openssl/1.1.1h")
```

Reply:
So for new recipes you are always using the latest version of a required library available in conan-center-index? Who cares about bumping the versions in the future?

Reply:
There is no (automatic) process for this (yet).
But we strive for new recipes using the most recent version of its dependencies.